meta-mender-core/classes/mender-setup-grub.inc

```diff
@@ -1,11 +1,5 @@
 EXTRA_IMAGEDEPENDS_append_mender-grub = " grub-efi"
 
-# The device number of the interface to load partitions from. This is normally
-# empty, in which case it is deduced from MENDER_STORAGE_DEVICE. Only use this
-# if the indexing of devices is different in Grub and in the Linux kernel.
-MENDER_GRUB_STORAGE_DEVICE ??= "${MENDER_GRUB_STORAGE_DEVICE_DEFAULT}"
```

Comment:
Note that this is a [documented variable](https://github.com/mendersoftware/mender-docs/search?q=MENDER_GRUB_STORAGE_DEVICE&unscoped_q=MENDER_GRUB_STORAGE_DEVICE), so it should probably be marked as deprecated.

I would also, in addition to your existing `Changelog: Title`, add:

```
Changelog: Deprecate and remove `MENDER_GRUB_STORAGE_DEVICE` variable.
```

Just to be explicit about it in the changelog (yes, you can have two changelog entries in one commit!).

Reply:
Updated changelog on its was as well as a mender-docs PR.

Reply:
Docs: https://github.com/mendersoftware/mender-docs/pull/894